source/server/syscfg_library.h

```diff
@@ -87,6 +102,11 @@ class SysCfgLibrary : public SysCfgLibraryInterface {
       NISysCfgFilterHandle filter_handle,
       const char* expert_names,
       NISysCfgEnumResourceHandle* resource_enum_handle);
+  using GetInstalledSoftwareComponentsPtr = NISysCfgStatus (*)(
```

Comment:
No Change Required.

I think these could all use the `using FunctionPtr = decltype(&Function)` pattern that the generated code uses.